wger/exercises/api/serializers.py

```diff
@@ -77,3 +68,30 @@ class MuscleSerializer(serializers.ModelSerializer):
     class Meta:
         model = Muscle
         fields = '__all__'
+
+class ExercisesDetailsSerializer(serializers.ModelSerializer):
```

Comment:
The ExercisesDetailsSerializer looks like a duplicate of ExerciseSerializer. What is it doing differently that ExerciseSerializer is not doing?

Reply:
I noticed that and I have made the corrections. Thanks.